core/backend/src/CloudSqlite.ts

```diff
@@ -576,14 +582,52 @@ export namespace CloudSqlite {
     promise: Promise<boolean>;
   }
 
+  /**
+   * Clean any unused deleted blocks from cloud storage. When a database is written, a subset of its blocks are replaced
```

Comment:
> When a database is written

"Written" seems like it means "created", is that the intent?
Written to? Updated?

Reply:
Hmm for a checkpoint, its created/written once. But I guess a database could be written to aka updated as well replacing teh subset of its blocks

Reply:
Changed it to `/**
   * Clean any unused deleted blocks from cloud storage. Unused deleted blocks can accumulate in cloud storage in a couple of ways:
   * 1) When a database is updated, a subset of its blocks are replaced by new versions, sometimes leaving the originals unused.
   * 2) A database is deleted with [[CloudContainer.deleteDatabase]]
   * In both cases, the blocks are not deleted immediately. Instead, they are scheduled for deletion at some later time.
   * Calling this method deletes all blocks in the cloud container for which the scheduled deletion time has passed.`